src/olympia/blocklist/models.py

```diff
@@ -363,16 +380,16 @@ def _split_guids_fake(cls, guids, v_min, v_max, filter_existing):
 
         # And then any existing block instances
         block_qs = Block.objects.filter(guid__in=guids).values_list(
-            'id', 'guid', 'min_version', 'max_version', 'kinto_id')
+            'id', 'guid', 'min_version', 'max_version', 'kinto_id', named=True)
```

Comment:
Do we need `named=True` here, or was that just for debugging purposes?

Reply:
it's needed - what it does is `.values_list` then returns a `namedtuple` rather than just a `tuple` so we can iterate over `x` and reference the properties as `x.id` rather than having to destructure `x` into it's properties (and having to name them again) or reference the `tuple` elements by index (`x[0]`).